src/test/java/org/openrewrite/staticanalysis/AddSerialAnnotationToserialVersionUIDTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.staticanalysis;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class AddSerialAnnotationToserialVersionUIDTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new AddSerialAnnotationToserialVersionUID());
+    }
+
+    @Test
+    void serialAnnotationAlreadyPresent() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+                             import java.io.Serializable;
+                             import java.io.Serial;
+                                         
+                             public class Example implements Serializable {
+                                 String var1 = "first variable";
+                                 @Serial
+                                 private static final long serialVersionUID = 1L;
+                                 int var3 = 666;
+                                 }
+                    """
```

Comment:
Could you have IntelliJ apply it's formatter? I think the shortcut is `Ctrl + Alt + L` on Windows too.

Reply:
I did use that but for some reason it garbles things. I borrowed test code from another recipe where it was formatted corrected. When using the formatter it got garbled. Will look into this to get it right.